src/index.ts

```diff
@@ -54,14 +55,15 @@ export default class BlueButton {
       throw new Error("clientSecret is required");
     }
 
-    if (!bbJsonConfig.callbackUrl) {
-      throw new Error("callbackUrl is required");
-    }
+    // if (!bbJsonConfig.callbackUrl) {
```

Comment:
Why is this commented out? Shouldn't callback url be required?

Reply:
uncommented